tests/pyccel/scripts/classes/class_variables.py

```diff
@@ -0,0 +1,15 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring, missing-class-docstring
+
+class A:
+    x : int
+    def __init__(self : 'A'):
+        self.x = 3
+        self.z : float = 10.0
+
+    def get_4(self : 'A'):
+        return 4
```

Comment:
Is the annotation of `self` mandatory or optional?

Reply:
Currently it is mandatory. See #1578